{"例外に関するシリアル化されたデータを含む例外状態オブジェクトが作成されたときに発生します":[131073,196609,262145,393217,655361,720897,786433,851969,440401921,454557697,456130561,456327169,458883073,460521473,461701121,463273985],"例外がシリアル化され":[131073,196609,262145,393217,655361,720897,786433,851969,440401921,454557697,456130561,456327169,458883073,460521473,461701121,463273985],"例外に関する追加のユーザー定義情報を提供する":[303104001,303431681,303955969,307298305,307691521,309329921,310640641,312672257,440401921,454557697,456130561,456327169,458883073,460521473,461701121,463273985]}